wuvt/static/js/live.js

```diff
@@ -105,7 +128,12 @@ function loadPage(path) {
     }).done(function(data) {
         updatePage($.parseHTML(data, document, true));
     }).fail(function(data) {
-        updatePage($.parseHTML(data.responseText, document, true));
+        if(data.status < 500) {
+            updatePage($.parseHTML(data.responseText, document, true));
+        } else {
+            $('title').text("Internal Server Error");
+            $('#content').html("<section><header><h2>Internal Server Error</h2></header><p>We apologize for the inconvenience. Please try your request again later.</p></section>");
```

Comment:
Another >80 char but I'm not sure if it's a big deal to you or not

Reply:
Actually I just noticed this is javascript so I'm not sure what their rule is. Ignore this one